spanner/spanner_snippets/spanner/spanner_directed_read_options.go

```diff
@@ -0,0 +1,104 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package spanner
+
+import (
+	"context"
+	"fmt"
+	"io"
+
+	"cloud.google.com/go/spanner"
+	sppb "cloud.google.com/go/spanner/apiv1/spannerpb"
+	"google.golang.org/api/iterator"
+)
+
+/*
+	Shows how to run a query with directed read options.
```

Comment:
issue: please reformat this comment. If the following information is available on a documentation page, link to that page here.

See these resources:
https://googlecloudplatform.github.io/samples-style-guide/#description
https://go.dev/doc/effective_go#commentary

Reply:
Comment refactored.